modules/Microsoft.Network/networkInterfaces/readme.md

```diff
@@ -316,6 +330,20 @@ module networkInterfaces './Microsoft.Network/networkInterfaces/deploy.bicep' =
           ],
           "name": "ipconfig01",
           "subnetResourceId": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/virtualNetworks/adp-<<namePrefix>>-az-vnet-x-001/subnets/<<namePrefix>>-az-subnet-x-001"
+        },
+        {
+          "applicationSecurityGroups": [
+            {
+              "id": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/applicationSecurityGroups/adp-<<namePrefix>>-az-asg-x-001"
+            }
+          ],
+          "loadBalancerBackendAddressPools": [
+            {
+              "id": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/loadBalancers/adp-<<namePrefix>>-az-lb-internal-001/backendAddressPools/servers"
+            }
+          ],
+          "name": null,
```

Comment:
As per previous comment, if we switch `!empty(ipConfiguration.name)` --> `contains(ipConfiguration, 'name')`. This won't be needed anymore. Also we can remove the `loadBalancerBackendAddressPools` and keep `applicationSecurityGroups` as this one is mandatory if the previous ipconfig references an ASG.

Reply:
The change would be at the `parameters.json` which then will get reflected here in the readme once its updated.

Reply:
makes sense! Will fix.